pkg/sensors/tracing/genericuprobe.go

```diff
@@ -759,7 +759,7 @@ func createMultiUprobeSensor(sensorPath string, multiIDs []idtable.EntryID, poli
 		regsMap := program.MapBuilderProgram("regs_map", load)
 		sleepableOffloadMap := program.MapBuilderProgram("sleepable_offload", load)
 		sleepableOffloadMap.SetMaxEntries(sleepableOffloadMaxEntries)
-		maps = append(maps, regsMap)
+		maps = append(maps, regsMap, sleepableOffloadMap)
```

Comment:
@olsajiri seems like in #4173 `sleepableOffloadMap` was missing for multi uprobe sensors.
I added the fix in its own commit (ready to drop it in case it was a desired behavior!)

Reply:
Do you mean opening a new PR with the 
> 		maps = append(maps, regsMap, sleepableOffloadMap)

patch?

Reply:
and https://github.com/cilium/tetragon/pull/4226 for the `sleepableOffloadMap` one.

Reply:
> the change for uretprobe below would not be necessary, thanks

Do you mean that i should not support `sleepable_offload` for uretprobe?